src/pages/SubmissionDetails.test.js

```diff
@@ -151,7 +151,7 @@ describe('Submission Details page', () => {
 
     await waitFor(() => {
       expect(getRuns).toHaveBeenCalledTimes(1)
-      expect(getRunsSets).toHaveBeenCalledTimes(1)
+      expect(getRunsSets).toHaveBeenCalledTimes(3)
```

Comment:
Out of curiosity, why is this being called 3 times now? 🤔

Reply:
In my most recent commit, this will be called twice now because it's called once in the `refresh()` function, and then called again when mounted. Is this too many? Should I try and consolidate?

Reply:
No I was just curious. Sounds good 👍